website/static/css/pages/wiki-page.css

```diff
@@ -319,3 +319,14 @@ select.form-control {
 .wiki-toolbar-icon:hover {
   background: rgba(51, 122, 183, 0.15);
 }
+
+.wiki-view-icon-container  /* this keeps the container of the view icon and dropdown responsive */
+{
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    -ms-transform: translateY(-50%);
+    -webkit-transform: translateY(-50%);
+    -moz-transform: translateY(-50%);
+
+}
```

Comment:
Minor, but please always include a newline at the end of a file.

<img width="137" alt="screen shot 2016-05-25 at 15 41 04" src="https://cloud.githubusercontent.com/assets/5659262/15553853/209488ba-228f-11e6-8a90-2ee810fde09d.png">